hw/drivers/crypto/crypto_k64f/src/arch/cortex_m4/mmcau_aes_functions.s

```diff
@@ -0,0 +1,941 @@
+#*******************************************************************************
```

Comment:
I assume license is ok for us to include this?

Reply:
Yes, BSD, it's missing the LICENSE header, I will add it or LICENSE file.

Reply:
Actually not so simple, but I'll review it and comment back here.